### ISSUE TITLE
Service not restarting after Force stop

### ISSUE THREAD

**Issue by SerhiiOst** [robertohuertasm/endless-service#11]
I've seen application restarting their service not only after reboot, but also after force stop. Any idea of how to do this?

**Comment by hpsaturn** [comment]
Yes, in Java and Kotlin it happen. OnTaskRemoved only is reached with a basic close app, with force stop the complete service don't works anymore. In previous versions, I think maybe Android 21 or less, with a simple Schedule Alarm the service return again, but with last API levels it doesn't works. Anyone has any idea?
